app/Services/SC/ScCoreService.cs

```diff
@@ -229,7 +229,8 @@ public async Task<ScAvailableTimesViewModel> LoadAvailableTimesFormAsync()
                 ConferenceLocationRegistryId = bookingInfo.BookingLocationRegistryId,
                 SelectedRegularTrialDate = bookingInfo.SelectedRegularTrialDate,
                 SelectedFairUseTrialDates = bookingInfo.SelectedFairUseTrialDates,
-                SessionInfo = bookingInfo
+                SessionInfo = bookingInfo,
+                RegistryContactNumber = GetRegistryContactNumber(bookingInfo.CaseRegistryId)
```

Comment:
I'm not 100% sure, but I think maybe this should be `TrailLocationRegistryId`?

Reply:
In these two contexts, it looks like the TrialLocationRegistryId will never be null, so I've updated it to that with no fallback